File: qr_test.py
import qrcode as qr
img = qr.make("https://www.youtube.com/watch?v=56_rgF058QI")
img.save("FG_2_YT.png") 



import qrcode 
from PIL import Image
import qrcode.constants

qr = qrcode.QRCode(version=1,
            error_correction=qrcode.constants.ERROR_CORRECT_H,
            box_size=10, border=3,)
qr.add_data("https://www.youtube.com/watch?v=56_rgF058QI")
qr.make(fit=True)
img = qr.make_image(fill_color = "red", back_color = "yellow")
img.save("FG_2_YT_2.png")

File: qr_test.js
# npm install qrcode
const QRCode = require('qrcode');
const fs = require('fs');

const url = 'https://www.youtube.com/watch?v=56_rgF058QI';
const outputPath = 'FG_2_YT.png';

QRCode.toFile(outputPath, url, function (err) {
  if (err) throw err;
  console.log('QR code saved to ' + outputPath);
});



const QRCode = require('qrcode');
const Jimp = require('jimp');

const url = 'https://www.youtube.com/watch?v=56_rgF058QI';

const options = {
  errorCorrectionLevel: 'H',
  margin: 3,
  color: {
    dark: '#FF0000', // Red color
    light: '#FFFF00' // Yellow color
  }
};



# npm install qrcode jimp

// Generate QR code
QRCode.toBuffer(url, options, (err, buffer) => {
  if (err) throw err;

  // Save the image
  Jimp.read(buffer, (err, image) => {
    if (err) throw err;
    image.write('FG_2_YT_2.png', (err) => {
      if (err) throw err;
      console.log('QR code saved to FG_2_YT_2.png');
    });
  });
});
